pkg/server/observer.go

```diff
@@ -60,12 +60,13 @@ func NewServer(
 	serviceCache *servicecache.ServiceCache,
 	payloadParser *parser.Parser,
 	maxFlows int,
+	eventQueueSize int,
```

Comment:
I'm not really comfortable with adding yet another parameter to `NewServer()` as the list is already quite long. Eventually, I think this should be refactored.

Reply:
yes. Options coming soon